app/frontend/src/components/designer/FloatButton.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div
+    :class="{ 'dir-rtl': true }"
```

Comment:
Is this supposed to always be true, or is it based on isRTL? (if always true, then no need to add isRTL to the computed)